app/src/main/AndroidManifest.xml

```diff
@@ -50,27 +57,48 @@
             <intent-filter>
                 <action android:name="android.accessibilityservice.AccessibilityService" />
             </intent-filter>
-            <meta-data android:name="android.accessibilityservice" android:resource="@xml/autofill_config" />
+
+            <meta-data
+                android:name="android.accessibilityservice"
+                android:resource="@xml/autofill_config" />
```

Comment:
Can we drop all the extra formatting?

Reply:
It's the default « autoformat » of Android studio. But ok. I will rebase to master and drop all extra formatting.